site.json

```diff
@@ -3139,6 +3139,15 @@
             "title": "JavaScript Demo: WeakMap.prototype.set()",
             "type": "js"
         },
+        "angle": {
+            "baseTmpl": "tmpl/live-css-tmpl.html",
+            "cssExampleSrc":
+                "../../live-examples/css-examples/css/angle.css",
+            "exampleCode": "live-examples/css-examples/angle.html",
+            "fileName": "angle.html",
+            "title": "CSS Demo: angle",
```

Comment:
The only comment I have is: should we make the title "CSS Demo: \<angle\>" to match the page title?